### PR TITLE
PLATUI-792: Replace serviceCode with serviceId for consistency with c…

### PR DESCRIPTION
**Pull request by matthewmascord**
…ontact-frontend

### CODE REVIEW

File: check-compatibility.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ if (!knownPrototypeKitNames.includes(consumerPackageJson.name)) {
 }
 
 const compatibility = {
-  '1.22': {
+  '1.23': {
     'prototype-kit': ['9.8', '9.7', '9.6', '9.5', '9.4', '9.3', '9.2', '9.1', '9.0', '9.9', '9.10']
```
Comment (timsb) [inline check-compatibility.js]: The prototype kit if on version 9.11.2, could you add relevant versions

File: src/components/report-technical-issue/template.njk [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% set language %}{% if params.language == 'cy' %}cy{% else %}en{% endif %}{% endset %}
-
+{% set serviceId = params.serviceId or params.serviceCode %}
```
Comment (timsb) [inline src/components/report-technical-issue/template.njk:2]: Just wondering does Nunjucks evaluate this left to right? If servceId and serviceCode is passed then does it just pick up serviceId?

**Review by timsb — CHANGES_REQUESTED** [review]

File: check-compatibility.js (hunk 1 above)
Comment (matthewmascord) [reply check-compatibility.js]: Thanks, updated now.

File: src/components/report-technical-issue/template.njk (hunk 2 above)
Comment (matthewmascord) [reply src/components/report-technical-issue/template.njk:2]: Good question. I've added a test for this edge case. serviceId takes precedence.

**Review by timsb — APPROVED** [review]
